agent/internal/container/container_runtime.go

```diff
@@ -1,46 +0,0 @@
-package container
-
-import (
-	"context"
-	"syscall"
-
-	"github.com/determined-ai/determined/agent/pkg/docker"
-	"github.com/determined-ai/determined/agent/pkg/events"
-	"github.com/determined-ai/determined/master/pkg/aproto"
-	"github.com/determined-ai/determined/master/pkg/cproto"
-
-	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/api/types/filters"
-)
-
-// ContainerRuntime is our interface for interacting with runtimes like Docker or Singularity.
-type ContainerRuntime interface {
```

Comment:
I kinda like the interface. I think its a lot harder to go the other way (that is put code behind a clean interface when it isn't) than it is to go from a clean interface to not having one.

So I think my preference would be to just leave this in here for now.